src/scheduler/check.h

```diff
@@ -226,33 +226,18 @@ schd_resource *zero_res(void);
 schd_resource *unset_str_res(void);
 
 /*
- *	find_correct_nodes - find the correct node_info and node partition
- *			     arrays to use for satisfing a job/resv
- *
- *
- *	  IN : sinfo - server associated with job/resv
- *	  IN : qinfo - queue associated with job (NULL if resv)
- *	  IN : resresv - the job/resv
- *	  OUT: ninfo_arr - the correct node array
- *	  OUT: nodepart - the correct node partition array
- *
- *	returns 1 on success or 0 on failure/error
- */
-int find_correct_nodes(status *policy, server_info *sinfo, queue_info *qinfo, resource_resv *resresv, node_info ***ninfo_arr, node_partition ***nodepart);
-
-/*
  *	get_resresv_spec - gets the correct select and placement specification
  *
  *	returns void
  */
 void get_resresv_spec(resource_resv *resresv, selspec **spec, place **pl);
 
 /*
- *	mark_resv_nodes_for_alter - gets the correct select and placement specification
+ *	reorder_nodes_set - reorder nodes in the superset to bring nodes in the subset to front.
  *
  *	returns void
  */
-void mark_resv_nodes_for_alter(node_info **unassoc_nodes, node_info **resv_nodes, int flag);
+void reorder_nodes_set(node_info **superset, node_info **subset);
```

Comment:
I wouldn't call this nodes_set.  There is something called 'node_set' in the scheduler currently.  Maybe move_subset_to_front() or something like that.

Reply:
This function will be removed as we discussed offline.